docs/docs/config/backlight.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Backlight Configuration
+sidebar_label: Backlight
+---
+
+See the [backlight feature page](/docs/features/backlight) for more details, including instructions for adding backlight support to a board.
```

Comment:
As of https://github.com/zmkfirmware/zmk/pull/911, all internal links now point to the file path rather than the URL. So for example this one should look like so:

```suggestion
See the [backlight feature page](../features/backlight.md) for more details, including instructions for adding backlight support to a board.
```

Linking to the document file path rather than the document URL ensures that the link resolves regardless of trailing slash config (#1249 describes the problems that this avoids).

Reply:
Links should be fixed now.